vm/src/builtins.rs

```diff
@@ -714,7 +714,7 @@ fn builtin_sum(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
 // builtin_vars
 // builtin___import__
 
-pub fn make_module(ctx: &PyContext) -> PyObjectRef {
+pub fn mk_module(ctx: &PyContext) -> PyObjectRef {
```

Comment:
I agree mk_module is more consistent, but make_module is a better name, since it does not contain abbreviations.

Reply:
Agreed. Okay to do a bunch of renaming in a separate PR?

Reply:
Yes, it is definitely an seperate PR.